### PR TITLE
WIP: enable debug on jenkinsfile

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ pipeline {
         SOCOK8S_DEVELOPER_MODE = "True"
         DEPLOYMENT_MECHANISM = "openstack"
         ANSIBLE_STDOUT_CALLBACK = "yaml"
+        ANSIBLE_DEBUG = "True"
```
Comment (evrardjp) [inline Jenkinsfile]: Do we really need this?  Things will become hardly readable.

I expect that ``ANSIBLE_VERBOSITY=2`` (or 1) would be enough for most cases

Comment (Itxaka) [reply Jenkinsfile]: This was just to catch the problem with CI jenkins jobs failing to reboot

Comment (evrardjp) [reply Jenkinsfile]: sounds good. We should have a look at pipelining too, and enable it by default in gates if it's not yet done.

**Comment by Itxaka** [comment]
not happening anymore so closing

**Comment by Itxaka** [comment]
hitting the same issue, so reopening to check if I can hit it with this PR

**Comment by jsuchome** [comment]
Looks unnecessary now

**Pull request closed without merging**